app/ui-flextab/client/tabs/userInfo.html

```diff
@@ -1,128 +1,3 @@
-								{{/if}}
-								{{#if createdAt}}
-									<div class="rc-user-info-details__item">
-										<label class="rc-user-info-details__label">{{_ "Created_at"}}</label>
-										<p class="rc-user-info-details__info">{{createdAt}}</p>
-									</div>
-									<div class="rc-user-info-details__item">
-										<label class="rc-user-info-details__label">{{_ "Last_login"}}</label>
-										<p class="rc-user-info-details__info">{{lastLogin}}</p>
-									</div>
-								{{/if}}
-								{{#if shouldDisplayReason}}
-									<div class="rc-user-info-details__item">
-										<label class="rc-user-info-details__label">{{_ "Reason_To_Join"}}</label>
-										<p class="rc-user-info-details__info">{{user.reason}}</p>
-									</div>
-								{{/if}}
-							{{/if}}
-							{{#if utc}}
-								<div class="rc-user-info-details__item">
-									<label class="rc-user-info-details__label">{{_ "Timezone"}}</label>
-									<p class="rc-user-info-details__info">{{userTime}} (UTC {{utc}})</p>
-								</div>
-							{{/if}}
-							{{#each customField}}
-								<div class="rc-user-info-details__item">
-									<label class="rc-user-info-details__label">{{_ label}}</label>
-									<p class="rc-user-info-details__info">{{value}}</p>
-								</div>
-							{{/each}}
-							<div class="rc-user-info-details__item">
-								{{#if services.facebook.id}} <p class="secondary-font-color"><i class="icon-facebook"></i><a href="{{services.facebook.link}}" target="_blank">{{services.facebook.name}}</a></p> {{/if}}
-								{{#if services.github.id}} <p class="secondary-font-color"><i class="icon-github-circled"></i><a href="https://www.github.com/{{services.github.username}}" target="_blank">{{services.github.username}}</a></p> {{/if}}
-								{{#if services.gitlab.id}} <p class="secondary-font-color"><i class="icon-gitlab"></i>{{services.gitlab.username}}</p> {{/if}}
-								{{#if services.google.id}} <p class="secondary-font-color"><i class="icon-gplus"></i><a href="https://plus.google.com/{{services.google.id}}" target="_blank">{{services.google.name}}</a></p> {{/if}}
-								{{#if services.linkedin.id}} <p class="secondary-font-color"><i class="icon-linkedin"></i><a href="{{services.linkedin.publicProfileUrl}}" target="_blank">{{linkedinUsername}}</a></p> {{/if}}
-								{{#if servicesMeteor.id}} <p class="secondary-font-color"><i class="icon-meteor"></i>{{servicesMeteor.username}}</p> {{/if}}
-								{{#if services.twitter.id}} <p class="secondary-font-color"><i class="icon-twitter"></i><a href="https://twitter.com/{{services.twitter.screenName}}" target="_blank">{{services.twitter.screenName}}</a></p> {{/if}}
-								{{#if services.wordpress.id}} <p class="secondary-font-color"><i class="icon-wordpress"></i>{{services.wordpress.user_login}}</p> {{/if}}
-							</div>
-						</div>
-					</div>
-				</section>
-			{{/with}}
-		{{/if}}
-	{{/if}}
-</template>
+	{{> UserInfoWithData onClose=onClose username=username rid=rid }}
```

Comment:
Why not render `UserInfoWithData` instead of `userInfo`? Why is `userInfo` an wrapper instead of the generated template for the component itself?